app/templates/components/toezicht/inzendingen-table.hbs

```diff
@@ -21,7 +21,13 @@
       {{/unless}}
       <td class="u-visible--tablet-portrait-up" style="max-width: 250px;">{{row.bestuurseenheid.naam}} <span class="u-hidden--tablet-landscape-up text-fade"><br>{{row.bestuurseenheid.classificatie.label}}</span></td>
       <td class="u-visible--tablet-landscape-up">{{row.bestuurseenheid.classificatie.label}}</td>
-      <td style="max-width: 250px;">{{row.besluitType.label}} <span class="u-hidden--tablet-portrait-up text-fade"><br>{{row.bestuurseenheid.naam}} {{row.bestuurseenheid.classificatie.label}}</span></td>
+      <td style="max-width: 250px;">
+        {{row.besluitType.label}}<br>
+        <span class="u-hidden--tablet-portrait-up text-fade">{{row.bestuurseenheid.naam}} {{row.bestuurseenheid.classificatie.label}}</span><br>
+        {{#if row.isRegulation}}
```

Comment:
No need to add the `{{#if}}` block here. If a regulationType is set, we want to show it.

Reply:
May be I have misunderstood this statement:

> for task DL-250 they want to display a subtype in the 'Type document' column if (and only if) the type of document == 'Reglementen en verorderingen' 
you can see a screenshot of what they want attached to story DL-115

Are you sure we don't need the if statement?

Reply:
The if-statement is most  important in the form where they have to fill in the regulationType. There we enforce they can only set a value for regulationType if besluitType == 'Reglement en verorderingen' . When displaying the inzendingen in the overview we shouldn't apply the if-statement anymore, because `row.regulationType.label` will only contain a value if besluitType == 'Reglement en verorderingen' because the input form forces it like that.